Source/CronosPlayUnreal/Private/PlayCppSdkActor.cpp

```diff
@@ -41,8 +54,13 @@ APlayCppSdkActor::APlayCppSdkActor() {
 // Called when the game starts or when spawned
 void APlayCppSdkActor::BeginPlay() { Super::BeginPlay(); }
 
-void convertSession(::com::crypto::game_sdk::WalletConnectSessionInfo src,
-                    FWalletConnectSessionInfo &dst) {
+auto convertSession(::com::crypto::game_sdk::WalletConnectSessionInfo src,
```

Comment:
does it need `auto` or could it be fixed to `FWalletConnectSessionInfo` ?

Reply:
i'll check